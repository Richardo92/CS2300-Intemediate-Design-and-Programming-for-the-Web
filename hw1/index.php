<!doctype html> 
<html> 
<?php
	define( 'URL_COMMON', 'https://info2300.coecis.cornell.edu/users/_demosp16/www/hw1_common/' );
?>
<head> 
	<meta charset="UTF-8">
	<title>Homework 1 JavaScript</title>
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
	<link rel="stylesheet" href="<?php echo URL_COMMON ?>css/style.css">
</head>
<body>
<div class="row">
	<div class="banner">
		<img src="<?php echo URL_COMMON ?>img/2000px-Star_Wars_Logo.svg.png" alt="logo">
	</div>
</div>

<div class="container"> 
    <div id="control-panel">
        <p> Control Panel </p> 
        <button id="border">Border</button><br>
        <button id="toggle">Toggle</button><br>
        <button id="addClass">Add Class</button><br>
        <button id="quotes">Load Quote</button><br>
        <button id="convert">Convert</button>
        <div>
            <h3>Test Rewrite</h3>
            Select a Movie: 
            <select id="rewrite_select">
                <option selected disabled>Choose here</option>
                <option value="0">I The Phantom Menace</option>
                <option value="1">II Attack of The Clones</option>
                <option value="2">III Revenge of The Sith</option>
                <option value="3">IV A New Hope</option>
                <option value="4">V The Empire Strikes Back</option>
                <option value="5">VI Return of The Jedi</option>
            </select>
            New Text: <input id="rewrite_text" type="text" />
            <button class="small_button" id="test_rewrite">Test</button>
        </div>
        <div>
            <h3>Search [Already Implemented] </h3>
            <input id="search" type="text" name="search"/>
        </div>
        <div>
            <h3>Replace</h3>
             Original: <input id="original" type="text" />
             New Text: <input id="newtext" type="text" />
            <button class="small_button" id="replace">Replace</button>
            
        </div>
    </div>
    <div id="input-panel">
    </div>


    <div id="movies-container"> 
	        <div class="movies"> 
               <img class="movie-poster" src="<?php echo URL_COMMON ?>img/episode1.jpg" alt="Episode 1">
                <ul>
                    <li>Title: Star Wars Episode I The Phantom Menace</li> 
                    <li>Release Date: May 19, 1999</li>
                    <li>Running Time: 133 minutes</li>
                </ul>
		        <a href="#" class="alert download btn">Free Movie Download</a>      
            </div> 
            <div class="movies"> 
               <img class="movie-poster" src="<?php echo URL_COMMON ?>img/episode2.jpg" alt="Episode 2">
                <ul>
                    <li>Title: Star Wars Episode II Attack of The Clones </li> 
                    <li>Release Date: May 16, 2002</li>
			        <li>Running Time: 142 minutes</li>
                </ul>
		        <a href="#" class="alert download btn">Free Movie Download</a>      
            </div> 
            <div class="movies"> 
               <img class="movie-poster" src="<?php echo URL_COMMON ?>img/episode3.jpg" alt="Episode 3">
                <ul>
                    <li>Title: Star Wars Episode III Revenge of The Sith</li> 
                    <li>Release Date: May 19, 2005</li>
                    <li>Running Time: 140 minutes</li>      
                </ul>
		        <a href="#" class="alert download btn">Free Movie Download</a>      
            </div> 
            <div class="movies"> 
               <img class="movie-poster" src="<?php echo URL_COMMON ?>img/episode4.jpg" alt="Episode 4">
                <ul>
                    <li>Title: Star Wars Episode IV A New Hope</li> 
                    <li>Release Date: May 25, 1977</li>
                    <li>Running Time: 125 minutes</li>    
                </ul>
		        <a href="#" class="alert download btn">Free Movie Download</a>      
            </div> 
           <div class="movies"> 
                <img class="movie-poster" src="<?php echo URL_COMMON ?>img/episode5.jpg" alt="Episode 5">
                <ul>
                    <li>Title: Star Wars Episode V The Empire Strikes Back</li> 
                    <li>Release Date: May 21, 1980</li>
                    <li>Running Time: 126 minutes</li>
                </ul>
                <a href="#" class="alert download btn">Free Movie Download</a>      

        </div> 
            <div class="movies"> 
                <img class="movie-poster" src="<?php echo URL_COMMON ?>img/episode6.jpg" alt="Episode 6">
                <ul>
                    <li>Title: Star Wars Episode VI Return of The Jedi</li> 
                    <li>Release Date: May 25, 1983</li>
                    <li>Running Time: 134 minutes</li>
                </ul>
                <a href="#" class="alert download btn">Free Movie Download</a>      
            </div>      
</div>
 <h1>Favorite Quotes</h1>
<div class="quotes"></div>    
             
 </div> 

<div id="footer"> 
		<p> Banner image is from <a href="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6c/Star_Wars_Logo.svg/2000px-Star_Wars_Logo.svg.png"> Wikimedia </a> </p>
      <p class="source"> All Movie Posters were taken from <a href="http://coverwhiz.com">CoverWhiz.com.</a></p>   
</div> 

    <script type="text/javascript" src="js/script.js"></script>
</body>
</html>