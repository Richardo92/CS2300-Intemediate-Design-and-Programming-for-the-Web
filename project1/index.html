<!DOCTYPE html>
  <head>
    <title>Personal Website</title>
    <link rel="stylesheet" href="css/style.css"/>
  </head>
  <body>
    <div class="board">

    </div>

    <div class="aboutMe">
      <a href="aboutMe.html">
        <div class="overlay">
           <p>About Me</p>
        </div>
      </a>  
    </div>
    
    <div class="resume">
      <a href="file/resume.pdf">    
        <div class="overlay">
           <p>Resume</p>
        </div>
      </a>
    </div>
    
    <div class="experience">
      <a href="experience.html">
        <div class="overlay">
           <p>Experience</p>
        </div>
      </a>
    </div>

    <div class="contact">
      <a href="contact.html">
        <div class="overlay">
           <p>Contact</p>
        </div> 
      </a>  
    </div>
    

    <div class="footer" style="margin-top:50px">        
        <div class="footer_address">
            <span style="font:Times New Roman">
                <h3>My Address</h3><br>
              Xinke Shidai, 1010 Nanhe Road<br>
              Chengguan District, Lanzhou 730010<br>
              Gansu, China<br>
            </span>
        </div>

        <div class="footer_contact">
            <span style="font:Times New Roman">
                <h3>My Contact</h3><br>
          <img src="img/phone.png">:(+86)13672001933<br>
          <img src="img/mail.png">:lidanni1993@yeah.net
            </span>
        </div>
                    
        <div class="footer_social">
            <span style="font:Times New Roman">
            <h3>My Social Account</h3><br>
            </span>
            <div>
                <a href="http://google.com/+"> <img src="img/google+.png"></a>
                <a href="http://www.facebook.com/"><img src="img/facebook.png"></a>
                <a href="http://www.linkedin.com/"><img src="img/linkedin.png"></a>
                <a href="http://twitter.com/"><img src="img/twitter.png"></a>
                <a href="http://youtube.com/"><img src="img/youtube.png"></a>
                <a href="http://instagram.com/"><img src="img/instagram.png"></a>
            </div>
        </div>
                    
        <div style="margin-left:600px; margin-top:170px">
            <br>
            <p>&copyCopyright 2016 Danni Li</p>
        </div>            
    </div>
  </body>
</html>