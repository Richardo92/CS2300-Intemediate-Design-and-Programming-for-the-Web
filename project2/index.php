<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
		<title>Job Wall</title>
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
		<?php 
			require "php/functions.php";
			require 'php/openFile.php';
			require 'php/readFile.php';
			require 'php/drawTable.php';
			require 'php/validateUser.php';
			require 'php/writeFile.php';
			require 'php/createFile.php';
			//add_versioned_file( 'js/job.js', 'JavaScript' );
			add_versioned_file( 'css/style.css', 'Style' );
		?>
	</head>

  <body>
  	<header>
		<nav id="menubar">
			<ul>
				<li><a href="index.php"><strong>Job List</strong></a></li>
				<li><a href="addJob.php"><strong>Add Job</strong></a></li>
				<li><a href="searchJob.php"><strong>Search Job</strong></a></li>
			</ul>
		</nav>
		<form method="POST" style="padding-top:15px">
			Username:<input class="login" type="text" name="username" placeholder="Please enter your username">
			Password:<input class="login" type="password" name="password" placeholder="Please enter your password">
			<input  type="submit" style="width:100px; height:35px" name="login" value="Login">
			<input  type="submit" style="width:100px; height:35px;" name="register" value="Register">
			<input  type="submit" style="width:100px; height:35px;" name="cancel" value="Login Out">
		</form>
	</header>

	<br><br><br><br>
	<h1>Job Wall</h1>
	<p id="welcome"></p>
	<?php
		// $file_pointer = openFile('status.txt', 'r');
		/* status variable */
		$isLogined = "false";
		$usn = "";
		$pwd = "";
		$counter = 0;
		

		/* open file, data.txt to read all info about jobs */
		$file_open = openFile('data.txt', 'r');
		/* read info */
		$jobs = readFileFromData($file_open);



	?>

		<!-- <form method="post"> -->
	<form method="post">
		<table border="1" style="width: 55%">
			<thead style="height:50px">
				<th style="width:200px">Company Name</th>
				<th style="width:150px">Job ID</th>
				<th style="width:200px">Location</th>
				<th style="width:150px">Salary</th>
				<th>Add To Cart</th>
			</thead>
			<tbody id="jobBody">
				<?php
					/* draw table depending on the data read from data.txt */
					drawTableWithAdd($jobs);
				?>
			</tbody>
		</table>
	</form>
	<br><br><br>
	<h1>My Cart</h1>
	<table border="1" style="width: 46%">
		<thead style="height:50px">
			<th style="width:200px">Company Name</th>
			<th style="width:150px">Job ID</th>
			<th style="width:200px">Location</th>
			<th style="width:150px">Salary</th>
		</thead>

		<tbody id="cartBody">
			<?php
				if (isset($_POST["login"])) {
					$username = $_POST["username"];
					$password = $_POST["password"];
					$file_open = openFile("usr.txt", 'r');
					if (!validateUser($file_open, $username, $password)) {
						echo "<script> alert('Failed to login. Invalid username or password'); </script>";
						exit;
					}

					$file_usr = openFile("usr_cart/usr_$username.txt", 'r');
					$user_jobs = readFileFromData($file_usr);
					drawTableWithoutAdd($user_jobs);

					$file_pointer = openFile('status.txt', 'w+');
					$line = trim("true")."\n";
					$line = $line.trim($username);
					writeFileFromBegin($file_pointer, $line);
				}
				else if (isset($_POST["register"])) {
					$username = $_POST["username"];
					$password = $_POST["password"];
					$username = trim($username);
					$password = trim($password);

					if ($username != "" && $password != "") {
						// add new username and password into usr.txt
						$file_create = openFile('usr.txt', 'a+');
						if (isUserExisted($file_create, $username)) {
							echo "<script> alert('User has already existed!'); </script>";
							exit;
						}
						$file_create = openFile('usr.txt', 'a+');
						$line = $username."\t".$password."\n";
						writeFileAtEnd($file_create, $line);
						createFileForUser("usr_cart/usr_$username.txt");
						/* set user state as log out */
						$file_pointer = openFile('status.txt', 'w+');
						$line = trim("false");
						writeFileFromBegin($file_pointer, $line);
					}
					else {
						echo "<script> alert('Failed to register! Invalid username or password'); </script>";
					}
				}
				else if (isset($_POST["cancel"])) {
					$file_pointer = openFile('status.txt', 'w+');
					$line = trim("false");
					writeFileFromBegin($file_pointer, $line);
				}
				/* update user state */
				$file_pointer = openFile('status.txt', 'r');
				while( ! feof( $file_pointer ) ) {
					$line = fgets( $file_pointer );
					if (empty($line) || $line === "" || $line === "\n")
						continue;
					if ($counter === 0) {
						$isLogined = trim($line);
						if ($isLogined === "false") {
							break;
						}
					}
					else if ($counter === 1) {
						$usn = $line;
					}
					$counter++;
				}
				
				for ($i = 0; $i < count($jobs); $i++) {
					if (isset($_POST["job$i"])) {
						if ($isLogined === "false") {
							break;
						}
						$file_pointer = openFile("usr_cart/usr_$usn.txt", 'a+');
						if (isJobExisted($file_pointer, $jobs[$i])) {
							print("<p>The job has already existed.</p>");
						}
						else {
							$file_pointer = openFile("usr_cart/usr_$usn.txt", 'a+');
							$line = implode("\t", $jobs[$i]);

		                	$line = "\n".$line;
		                	writeFileAtEnd($file_pointer, $line);
						}
						
		                $file_pointer = openFile("usr_cart/usr_$usn.txt", 'r');
						$lines = readFileFromData($file_pointer);
						drawTableWithoutAdd($lines);
						break;
					}
				}
				
				if ($isLogined === "true") {
					echo "<script> document.getElementById('welcome').innerHTML = 'Welcome, $usn'; </script>";
				}
				else {
					echo "<script> document.getElementById('welcome').innerHTML = 'You are currently a guest. Please login or register ASAP.'; </script>";
				}
				//print("<p>No Login</p>");
			?>
		</tbody>
	</table>

	



  </body>

</html>
