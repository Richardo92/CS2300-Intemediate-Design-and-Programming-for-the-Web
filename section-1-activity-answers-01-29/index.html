<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<link rel="stylesheet" type="text/css" href="css/styles.css">
	<title>CS/INFO 2300 - Section 1</title>
</head>
<body>
	<h1>CS/INFO 2300 - Section 1 Activity</h1>
	<em>
		Instructions: Implement the following prompts using the external stylesheet "styles.css" 
		(no inline styling) and your knowledge of CSS3 and HTML5. There should be no need to edit
		or change the HTML: only use CSS to make changes to the page. Dont be afraid to ask your TAs and even
		your neighbors if you don't know how to do something or need a refresher!
	</em>

	<h3>1. Change the color of the following text to "aquamarine" (DO NOT use names for color values, only hex codes and rgb--find them <a class ="regular" href="http://www.w3schools.com/colors/colors_names.asp" target="_blank">here</a>).</h3>
	<span id="p1">Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum.</span>

	<h3>2. Add a dashed underline that is orange and 2px in thickness to the following word.</h3>
	<span id="p2">ITHACA</span>

	<h3>3. Turn the rectangle into a square with sides of length 100px and color red.</h3>
	<div class="rectangle" id="p3"></div>

	<h3>4. Give the rectangle dimensions of 2:3 and a photo background using the image "doge.jpg" found under the "img" folder--make sure the 
	background covers the entire area of the box.</h3>
	<div class="rectangle" id="p4"></div>

	<h3>5. Center the square horizontally on the page, convert it into a circle, and give it a dotted border with color "dark slate gray" and thickness of 5px</h3>
	<div id="p5"></div>

	<h3>6. Using "doge-1.jpg" under "img," create a circle picture placeholder with size 200px, and border of 4px + color "Antique White". Make sure to center 
	the face of the dog in the center of the circle. HINT: use background-position to find the right placement and background-size to zoom into the dog's face. Fiddle with your browser's inspector until you see the right position / size for the image (Chrome's inspector works best).</h3>
	<div class="rectangle" id="p6"></div>

	<h3>7. Create a simple horizontal navigation bar from this unordered list with the following specifications: a black background, white font. Make sure there is at least 20px of padding between each menu item and to remove the bullets.</h3>
	<ul id="p7">
		<li><a href="/">Home</a></li>
		<li><a href="/">About</a></li>
		<li><a href="/">Products</a></li>
		<li><a href="/">Contact</a></li>
	</ul>

	<h3>8. On hover over the entire text, change the last "p" text block font color to blue. Also, change the cursor to the "hand" gesture on hover. Again, do not change the HTML, this should be done purely through CSS.</h3>
	<div id="p8">
		<p>Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit.</p>
		<p>Nullam quis risus eget urna mollis ornare vel eu leo.</p>
		<p>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.</p>
		<p>Donec sed odio dui.</p>
	</div>

	<h3>9. Convert the three rectangles into squares of side 200px and center them evenly, horizontally. 
	Their color should be changed to "Yellow Green." Remember: if multiple objects have the same CSS attributes, GROUP THEM together when you assign attributes (Google DRY - Don't Repeat Yourself)</h3>
	<div id="p9">
		<div class="rectangle" id="one"></div>
		<div class="rectangle" id="two"></div>
		<div class="rectangle" id="three"></div>
	</div>

	<h3>10. Convert the boxes into cocentric circles (circles within each other that share the same center). The outer circle should be black with a size of 300px and the inner circle should be white with a size of 200px.</h3>
	<div id="p10">
		<div id="outer">
			<div class="rectangle" id="inner"></div>
		</div>
	</div>

</body>
</html>