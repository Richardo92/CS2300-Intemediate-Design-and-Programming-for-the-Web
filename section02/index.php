<!DOCTYPE html>
<html>
    <head>
        <meta charset="UTF-8">
        <title>INFO / CS 2300 Section 2</title>
        <style type="text/css">
            html, body {
                margin: 0;
                padding: 0;
            }
            html {
                background-color: black;
            }
            body {
                background-color: #ecce89;
                font-family: 'Segoe UI', Frutiger, 'Frutiger Linotype', 'Dejavu Sans', 'Helvetica Neue', Arial, sans-serif;
                margin: auto;
                padding: 10px;
                width: 80%;
            }

            #super-bowl-img {
                display: block;
                margin: 0 auto;
            }

            h1, #super-bowl-img-src {
                text-align: center;
            }
            .player-info {
                align-items: center;
                display: flex;
                height: 70px;
            }
            .team-logo {
                height: 70px;
                width: 70px;
            }
            a {
                color: deepskyblue;
            }
        </style>
    </head>
    <body>
        <h1>PHP SuperBowl</h1>
        <img src="https://mgtvwbtw.files.wordpress.com/2016/01/super-bowl-50-fs.png?w=650" id="super-bowl-img" />
        <div id="super-bowl-img-src">
            <a href="http://wbtw.com/2016/01/25/poll-who-will-win-super-bowl-50/">Image Source</a>
        </div>
        <p>Welcome to the greatest PHP review there ever was. In honor of this weekend's game, put the team on yo back and fill out the missing (or incomplete) PHP code to fix the form and display lists of the players on each team.</p>
        <!-- Self submitting form - no 'action' attribute needed -->
        <form method="POST">
            If you wish to add a player to the Panthers roster, select "Carolina Panthers" from the below menu.  If you wish to add a player to the Broncos roster, select "Denver Broncos".<br>
            <input type="text" name="newPlayer" />
            <select name="newTeam">
                <option value="Carolina Panthers">Carolina Panthers</option>
                <option value="Denver Broncos">Denver Broncos</option>
            </select>
            <input type="submit" name="submit" value="Add Player" />
        </form>

        <?php
            # The current associative array of players; don't edit
            $players = array(
                "Andre Caldwell" => "Denver Broncos",
                "Kawann Short" => "Carolina Panthers",
                "Peyton Manning" => "Denver Broncos",
                "Cam Newton" => "Carolina Panthers",
                "Demaryius Thomas" => "Denver Broncos",
                "Greg Olsen" => "Carolina Panthers"
            );
            # First down: Fix the following 'if' statement so that when submit
            # has been clicked, you get and store the POST values. What PHP
            # function do you know that does this?
            /*
            * ===============================================================================
            * EDIT BELOW THIS LINE ONLY; UNCOMMENT EACH SECTION SO THAT IT FUNCTIONS PROPERLY
            * ===============================================================================
            */
            
            if(isset($_POST["submit"])) {
                # Uncomment when you have fixed if-statement above
                # Next week, weâ€™ll start using something more secure than $_POST but OK for today
                $newPlayer = $_POST["newPlayer"];
                $newTeam = $_POST["newTeam"];
                # Second down: if newPlayer and newTeam are set and newPlayer matches the regular expression, then add it to the array to be printed
                if(isset($_POST["newPlayer"]) && isset($_POST["newTeam"]) && preg_match("/[a-zA-Z ]+/", $newPlayer)) {
                    # For the last preg_match() function, allow only letters (uppercase or lowercase) or spaces
                    # Remember that there are two arguments for preg_match: the pattern and the string to test
                    # Uncomment the below line when ready, and add the new player to the $players array
                    $players[$newPlayer] = $newTeam;
                }
                else {
                    echo "<h3>Please enter a valid player name for it to be added!</h3>";
                }
            }
        ?>

        <h3>Print the list of players using a foreach loop</h3>

        <?php
            /* Third and goal: Fill in the foreach parameters and the if statement, then uncomment this block */
            foreach($players as $playerName => $playerTeam) {
                # Print out the key and value, as in "Peyton Manning plays for the Denver Broncos".
                # We've gotten you started by adding the div and appropriate class
                # Fix the if statement so that the appropriate logo prints.
                # (Looking at the next line you'll realize that the 'if' should check if the team is Denver.)
                echo "<div class='player-info'>";
                if ($playerTeam == "Denver") {
                    $img = "http://prod.static.broncos.clubs.nfl.com/nfl-assets/img/gbl-ico-team/DEN/logos/home/large.png";
                }
                else {
                    $img = "http://prod.static.panthers.clubs.nfl.com/nfl-assets/img/gbl-ico-team/CAR/logos/home/large.png";
                }
                echo "$playerName plays for the $playerTeam: <img src='$img' class='team-logo' alt='$playerTeam Logo' />";
                echo "</div>";
            }
            # Touchdown! If there's time left, go for the two-point conversion.
        ?>


        <h3>Recreate the above using a function</h3>

        <?php
            # Two point conversion: define function here -- there should be two arguments.
            function printName(){
                # Use the content for this function from the above foreach loop, but NOT the actual loop.
                # You will loop through each array element below, and execute the function for each array element
                # in the context of the foreach loop.

            }


            # Call the function using a foreach (similar setup to before) loop
            
            /*foreach () {
                
            }
            */
            # Great job! You scored a TD and punched the two-point conversion in. Enjoy the Superbowl this weekend!
        ?>
        <h1><a href="http://www.cc.com/video-clips/qs3r6w/the-colbert-report-logo-restrictions-for-the-super-bowl">Have fun at the Superb Owl!</a></h1>
    </body>
</html>