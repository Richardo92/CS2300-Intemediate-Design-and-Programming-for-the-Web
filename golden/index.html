<!DOCTYPE HTML>
<html>
	<head>
		<meta charset="UTF-8">
		<script src="jquery-1.12.3.min.js"></script>
		<script language="javascript" type="text/javascript">
			$(document).ready(function(){ 
				$(".header").load("header.php"); 
			}); 
		</script>
	</head>
	<body>

		<div class = "header">
<!-- 			load the tltle and navigation bar -->
		</div>

<!-- 		body -->
		<div>
		<h4>
			<div class="intro" style="float:left; padding-left:50px">
				<iframe width="500" height="281" scrolling="no" frameborder="0" id="814670336" name="814670336" allowtransparency="true" class="igm" src="//mj89sp3sau2k7lj1eg3k40hkeppguj6j-a-sites-opensocial.googleusercontent.com/gadgets/ifr?url=http://www.gstatic.com/sites-gadgets/iframe/iframe.xml&amp;container=enterprise&amp;view=default&amp;lang=en&amp;country=ALL&amp;sanitize=0&amp;v=1cbf636963b6b8f8&amp;libs=core&amp;parent=https://sites.google.com/site/gold4students/#up_scroll=auto&amp;up_iframeURL=//player.vimeo.com/video/93194386&amp;st=e%3DAIHE3cCYR0Y8DrVxJ63BN4opoqnqfimnG7wPFh2ziir%252Fb9s61xTj7oahxloRrbdlQGUOv4%252B5CiAYlR2gTaefgz1moRzfzZ5Sd4u%252BilvSpEQRHu0izTNQOuC9qA7N7ToFYvzXoHhnjXBk%26c%3Denterprise&amp;rpctoken=5351319116724919140"></iframe>
			</div>
			<div style="float:right; width:500px; padding-right:200px">
				<p><a href = "https://soundcloud.com/whcu-radio/20160129martykaminsky">WHCU interview</a>
					with GO's founder, Marty Kaminsky 1-29-2016<br>
					Golden Opportunity (GO) is a not-for-profit tutoring program providing academic support for elementary and middle school students in the Ithaca City School District.<br>
					GO tutors are primarily retired teachers who are carefully matched with students to create an ideal fit. The selected youngsters have been referred by their teachers. Dedicated tutors work with their students throughout their elementary school years--most continuing into middle school.</p>
			</div>

			<div style="clear: left;"/> 
		</h4>
		<br>

			<h4> 
				<div style="float:left; padding-left:50px">
					<img src = "img/john.jpg"/>
				</div> 
				<div style="float:right; width:500px; padding-right:250px"> 
					<br><br>
					<p>The continuity of tutor/student relationship, along with the trust, insight, and compassion established between tutor and student over a period of years, make this program unique and effective.<p>
				</div> 
				<div style="clear: left;"/> 
			</h4>


			<h4> 
				<div style="float:left; width:500px; padding-left:50px"> 
					<br><br>
					<p>Retired teachers bring years of professional expertise to their students, accelerating learning progress. GO provides 60 hours per school year of direct one-on-one tutoring for each student during after school hours. Research shows such relationships foster student self-esteem and academic success.</p>
				</div> 
				<div style="float:right; padding-right:250px">
					<img src = "img/dictionary.jpg">
				</div> 
				<div style="clear: left;"/> 
			</h4>

			<br><br><br><br><br><br><br><br><br><br>
			<!-- <div>
				<p>Retired teachers bring years of professional expertise to their students, accelerating learning progress. GO provides 60 hours per school year of direct one-on-one tutoring for each student during after school hours. Research shows such relationships foster student self-esteem and academic success.</p>
				
			</div> -->
<!-- 				footer -->
			<div style="text-align: center">
				<p>Photos courtesy of Jo Todd unless otherwise ascribed</p>
			</div>
			<br>
		</div>

		<script src="js/bootstrap.min.js"></script>
	</body>
</html>